.github/workflows/ci.yml

```diff
@@ -103,11 +99,21 @@ jobs:
     name: "Docker Arm64 build"
```

Comment:
Do we want this in the `ci` workflow where the other jobs are for testing? And here we are producing images to be used in local development for sentry. This also takes 45+ minutes, which is way too long imo for CI. It might make more sense to have this in its own workflow that runs only on pushes to master.

Reply:
I've fixed the caching and it now takes less than 15 minutes.